### PR TITLE
(deprecated) Backporting to 42 - New content for GPG encrypted Salt Pillars

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Action item to document encrypted Salt pillars was the outcome of research in https://github.com/SUSE/spacewalk/issues/14613.

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default) https://github.com/uyuni-project/uyuni-docs/pull/1448
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/14853

### CODE REVIEW

File: modules/salt/pages/salt-gpg-encrypted-pillars.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+----
+echo 'gpg_keydir: /etc/salt/gpgkeys' >/etc/salt/master.d/gpg-pillar.conf
+systemctl reload-or-restart salt-master
+----
+
+
+== Use GPG for encrypting Pillar secrets
+
+Salt GPG renderer decrypts GPG encrypted contents that are ASCI-armored. 
+To use the GPG renderer in a Pillar YAML file, change
```
Comment (keichwa) [inline modules/salt/pages/salt-gpg-encrypted-pillars.adoc:49]:
```suggestion
To use the GPG renderer in a pillar YAML file, change 
```
Later you use this variant.

File: modules/salt/pages/salt-gpg-encrypted-pillars.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
++
+----
+echo 'gpg_keydir: /etc/salt/gpgkeys' >/etc/salt/master.d/gpg-pillar.conf
+systemctl reload-or-restart salt-master
+----
+
+
+== Use GPG for encrypting Pillar secrets
+
+Salt GPG renderer decrypts GPG encrypted contents that are ASCI-armored.
```
Comment (keichwa) [inline modules/salt/pages/salt-gpg-encrypted-pillars.adoc:48]:
```suggestion
Salt GPG renderer decrypts GPG encrypted contents that are ASCII-armored. 
```

**Review by keichwa — APPROVED** [review]
Maybe, I overlooked those when merging into master `:(`

**Comment by 0rnela** [comment]
New PR: https://github.com/uyuni-project/uyuni-docs/pull/1467

**Pull request closed without merging**